rwatch/ui/action_menu.c

```diff
@@ -73,7 +73,7 @@ ActionMenuItem * action_menu_level_add_action(ActionMenuLevel * level, const cha
     new_item->cb = cb;
     new_item->action_data = action_data;
     
-    if (level->capacity == level->count)
+    if (*level->capacity == level->count)
```

Comment:
kaboom. See comments above about this comparing integers and pointers (no way we should be derefing here)